src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -52,32 +53,30 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 userSelectNodeSymbol.Kind == SymbolKind.Event ||
                 userSelectNodeSymbol.Kind == SymbolKind.Field)
```

Comment:
thi code seems to be duplicating code i've seen in the PR where we check the validity of symbols.  won't we have already filtered to this set yet?  so why do we need these checks again?